libr/search/search.c

```diff
@@ -427,7 +406,9 @@ R_API RList *r_search_find(RSearch *s, ut64 addr, const ut8 *buf, int len) {
 
 /* --- keywords --- */
 R_API int r_search_kw_add(RSearch *s, RSearchKeyword *kw) {
-	if (!kw) return false;
```

Comment:
Now `	if (!kw || !kw->keyword_length) {`

I believe the call site should do the check, not here.